Bestsellers-section

Kateryna4213 opened the pull request.

greatberser committed:
done newsletter (checkbox in progress))

greatberser committed:
remove extra codes

greatberser committed:
Update subscription.css

greatberser committed:
changing the font-size on each screen, changed color checkbox, error-color inputs, added :focus on checkbox

greatberser committed:
update html&css subscription

greatberser committed:
changed padding-bottom on tablet, input border changes even during invalid info, added transition for error icon and border input. reworked section image

greatberser committed:
changed all padding, add name to alt

greatberser committed:
added label with aria-label, changed form margin, changed h3 to h2, changed custom checkbox

Kateryna4213 committed:
Bestsellers-section

Kateryna4213 committed:
Bestsellers-section

TigranKh21 committed:
Our products html+css

TigranKh21 committed:
Our products html+css

TigranKh21 committed:
Our products html+css

TigranKh21 committed:
html is changed (div's replaced by ul and li)

TigranKh21 committed:
html is changed (div's replaced by ul and li)

TigranKh21 committed:
html is changed (div's replaced by ul and li)

YuriiM-Owl reviewed (changes requested):
секція:
в тебе не адаптовані відступи між секціями, ти задала padding: 80px 0, хоча за макетом вони інші на мобільних пристроях і змінюються на планшетному брейкпоінті

заголовок секції:
зміна марджина відбувається на планшетному брейкпоінті, а не на десктопному

до img всередині picture ліпше вказувати зображення, яке має найбільшу ширину серед 1х, що прописані в твоїх source (в твоєму випадку це планшетне, а не мобільне)

за ТЗ назви меблів повині бути виконані як заголовки (h4), але стилізовані під текст

зображення, в тому числі весь тег picture, краще обгортати у додаткову div  обгортку і стилізувати відступи і інше вже через неї, а не напряму через прописування марджинів (чи будь-яких інших стилів) до img, чи picture

YuriiM-Owl committed:
Merge pull request #1 from YuriiM-Owl/newsletter

Newsletter

YuriiM-Owl committed:
corrected checkbox label

corrected adaptive behavior of font-size in .checkbox-container .form-label

YuriiM-Owl committed:
add class with alt styles for h4

TigranKh21 committed:
correction styles according team-lead comments

TigranKh21 committed:
correction styles according team-lead comments

TigranKh21 committed:
Merge branch 'Tygran-Our-products' of https://github.com/YuriiM-Owl/minimal-landing-page into Tygran-Our-products

TigranKh21 committed:
class local-title-alt font-size was set

TigranKh21 committed:
class local-title-alt font-size was set

YuriiM-Owl committed:
update .local-title-alt media

TigranKh21 committed:
Merge branch 'Tygran-Our-products' of https://github.com/YuriiM-Owl/minimal-landing-page into Tygran-Our-products

TigranKh21 committed:
<p> to <h4> was changed

YuriiM-Owl committed:
Merge pull request #8 from YuriiM-Owl/Tygran-Our-products

Our products html+css

Kateryna4213 committed:
Changes-css-html

Changed paddings in sections and swithced <p> to <h4>

Kateryna4213 committed:
Changes-css-html

Changed paddings in sections and swithced <p> to <h4>

Kateryna4213 committed:
changes-css

Kateryna4213 committed:
changes-css

Kateryna4213 committed:
changes-css

Kateryna4213 committed:
changes-css

Kateryna4213 committed:
Merge branch 'Bestsellers' of https://github.com/YuriiM-Owl/minimal-landing-page into Bestsellers

Kateryna4213 committed:
Changes-html-css

Kateryna4213 committed:
css + html changes

the main item of the <ul> is now a separated block (<div>)
Implemented requiered changes from pull-request and slack

YuriiM-Owl reviewed (changes requested):
Заголовок секції: нижній марджин не відповідає макету на планшетній версії (30px)

.bestsellers-item-display-none {display: none;} краще перенести в медіа-запит max-width:767 , тоді не потрібно давати display:block в медіа min-width:768

нижньому відступ у картинок на десктопі повернути зазначити за макетом, а висоту блоків вирівняти за допомогою лайн-хейту

видалити клас .text у заголовків h4, його стилі включені у .local-title-alt

Kateryna4213 committed:
Css + html final changes

Fixed small css issues due to pull request's requirements
